examples/tutorials/data/hawc.py

```diff
@@ -0,0 +1,268 @@
+# Check setup
+# -----------
+from gammapy.utils.check import check_tutorials_setup
+from gammapy.visualization import plot_theta_squared_table
+
+check_tutorials_setup()
+
+######################################################################
+# Chose which estimator we will use
+
+energy_estimator = "NN"
+
+
+######################################################################
+# A useful way to organize the relevant files are the index tables. The
+# observation index table contains information on each particular observation,
+# such as the run ID. The HDU index table has a row per
+# relevant file (i.e., events, effective area, psf…) and contains the path
+# to said file.
+# The HAWC data is divided into different event types, classified using
+# the fraction of the array that was triggered by an event, a quantity
+# usually referred to as "fHit". These event types are fully independent,
+# meaning that an event will have a unique event type identifier, which
+# is usually a number indicating which fHit bin the event corresponds to.
+# For this reason, a single HAWC observation has several HDU index tables
+# associated to it, one per event type. In each table, there will be
+# paths to a distinct event list file and IRFs.
+# In the HAWC event data release, all the HDU tables are saved into
+# the same FITS file, and can be accesses through the choice of the hdu index.
+
+
+######################################################################
+# Load the tables
+
+data_path = "$GAMMAPY_DATA/hawc/crab_events_pass4/"
+hdu_filename = f"hdu-index-table-{energy_estimator}-Crab.fits.gz"
+obs_filename = f"obs-index-table-{energy_estimator}-Crab.fits.gz"
+
+# there is only one observation table
+obs_table = ObservationTable.read(data_path + obs_filename)
+
+# we read the hdu index table of fHit bin number 6
+fHit = 6
+hdu_table = HDUIndexTable.read(data_path + hdu_filename, hdu=fHit)
+
+
+######################################################################
+# From the tables, we can create a Datastore
+
+data_store = DataStore(hdu_table=hdu_table, obs_table=obs_table)
+
+data_store.info()
+
+
+######################################################################
+# There is only one observation
+
+obs = data_store.get_observations()[0]
+
+######################################################################
+# Select and peek events
+
+obs.events.peek()
+
+
+######################################################################
+# Peek the energy dispersion
+
+obs.edisp.peek()
+
+######################################################################
+# Peek the psf
+obs.psf.peek()
+
+######################################################################
+# Peek the background for one transit
+plt.figure()
+obs.bkg.reduce_over_axes().plot(add_cbar=True)
+
+######################################################################
+# Peek the effective exposure for one transit
+plt.figure()
+obs.aeff.reduce_over_axes().plot(add_cbar=True)
```

Comment:
The background and aeff are per transit, but the psf or edisp are not?

Reply:
Everything is for one transit, but the PSF and EDisp don't depend on the number of transits, whereas the effective exposure and the background do.

Reply:
yes indeed! stupid question, sorry